README.md

```diff
@@ -12,12 +12,18 @@
 <img src="plugin/src/main/webapp/img/logo-head.svg" width="192">
 
 
+- [Introduction](##introduction)
+- [Examples](##examples)
+- [Initial Configuration](##initial-configuration)
 - [Handling Secrets](./docs/features/secrets.adoc)
 - [Exporting configurations](./docs/features/configExport.md)
 - [Validating configurations](./docs/features/jsonSchema.md)
 - [Triggering Configuration Reload](./docs/features/configurationReload.md)
-
-
+- [Installing plugins](##installing-plugins)
+- [Supported Plugins](##supported-plugins)
+- [Adding JCasC support to a plugin](##adding-jCasC-support-to-a-plugin)
+- [Configuration-as-Code extension plugins](##configuration-as-Code-extension-plugins)
+- [Jenkins Enhancement Proposal](##jenkins-enhancement-proposal)
```

Comment:
Probably no longer relevant, but we can keep it

Reply:
Thank you for the review